src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -72,18 +71,38 @@ $card-left-offset: 40px;
         justify-content: center;
     }
 
+    .bcs-task-status-container {
+        margin-top: 8px;
+    }
+
     .bcs-task-status {
         align-items: center;
+        color: $bdl-neutral-02;
         display: flex;
-        margin-top: 18px;
+        font-weight: bold;
+
+        .bcs-task-status-item:first-child {
+            margin-left: 5px;
```

Comment:
May be missing something, but why does it need :first-child? Can the item itself have the padding?

Reply:
The `In Progress` status doesn't have an icon with it, so either the icon or the status message needs the padding

Reply:
We can change this in a follow-up